src/app/navbar/navbar.component.scss

```diff
@@ -0,0 +1,54 @@
+$header-background: #466bc1;
+$button-hover-color: #3859a5;
+$font-black: black;
+
+.profilepic{
+		height:100%;
+		width: 100%;
+		padding: 15px;
+
+}
+
+.sidebarheader{
+    background: $header-background;
+    padding: 4px;
+
+}
+
+.profile{
+		text-align: -webkit-center;
+		font-size: 20px;
+}
+
+.sidebar{
```

Comment:
this design is VERY mobile-like/heavy. you don't typically see this type of layout for desktop web interfaces.
i know you are following the UI mock, which was designed for mobile in first place. thing is, i'm not sure if it makes sense for us to strictly follow it